### PR TITLE
Support longer UserName

### PR DESCRIPTION
**Pull request by mmpei**
Signed-off-by: peimingming

the fix(commit 571384cdb290be980616aeb86f5e7d34b6f13492) for this issue has some problem.
and limit length of user name to 80 in UI is reasonable, for it will look strange from UI if length is too long.
and user name length to 255 when creating by API. https://github.com/goharbor/harbor/issues/6086

### CODE REVIEW

**Comment by mmpei** [comment]
@steven-zou

File: src/portal/src/app/shared/new-user-form/new-user-form.component.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
             <div class="form-group form-group-override">
                 <label for="username" class="required form-group-label-override">{{'PROFILE.USER_NAME' | translate}}</label>
                 <label for="username" aria-haspopup="true" role="tooltip" class="tooltip tooltip-validation tooltip-md tooltip-bottom-left" [class.invalid]='getValidationState("username")'>
-                    <input type="text" required pattern='[^"~#$%]+' maxLengthExt="255" #usernameInput="ngModel" name="username" [(ngModel)]="newUser.username" id="username" size="30" 
+                    <input type="text" required pattern='[^"~#$%]+' maxLengthExt="80" #usernameInput="ngModel" name="username" [(ngModel)]="newUser.username" id="username" size="30"
```
Comment (ninjadq) [inline src/portal/src/app/shared/new-user-form/new-user-form.component.html]: maxLengthExt should be 255

File: src/portal/src/i18n/lang/en-us-lang.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
     },
     "TOOLTIP": {
         "EMAIL": "Email should be a valid email address like name@example.com.",
-        "USER_NAME": "Cannot contain special characters and maximum length should be 20 characters.",
+        "USER_NAME": "Cannot contain special characters and maximum length should be 80 characters.",
```
Comment (ninjadq) [inline src/portal/src/i18n/lang/en-us-lang.json]: max length should be 255

File: src/portal/src/i18n/lang/zh-cn-lang.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
     },
     "TOOLTIP": {
         "EMAIL": "请使用正确的邮箱地址，比如name@example.com。",
-        "USER_NAME": "不能包含特殊字符且长度不能超过20。",
+        "USER_NAME": "不能包含特殊字符且长度不能超过80。",
```
Comment (ninjadq) [inline src/portal/src/i18n/lang/zh-cn-lang.json]: the same as above

**Comment by mmpei** [comment]
@ninjadq  done, please check.

**Comment by clouderati** [comment]
@steven-zou ptal and merge if it lgty; thanks!

**Pull request merged**